backend/controllers/groups-controllers.js

```diff
@@ -0,0 +1,199 @@
+const express = require("express");
+const Group = require("../database/models/groups");
+const GroupClaims = require("../database/models/group_claims");
+const AdvancedSettings = require("../database/models/advanced_settings");
+const { v4: uuidv4 } = require("uuid");
+const { Op } = require("sequelize");
+const models = require("../database/models");
+
+const createGroup = async (req, res) => {
+  const group_id = uuidv4();
+  const groupName = req.body.name;
+  const creationDate = req.body.creationDate;
```

Comment:
what if creationDate = 03/08/2020 or 2020-08-03 9:48:23 ? you should have a data format and check if you get data in this format, otherwise return 400 Bad Request